src/java/main/org/apache/zookeeper/version/util/VerGen.java

```diff
@@ -93,6 +93,14 @@ public static void generateFile(File outputDir, Version version, int rev, String
         } catch (IOException e) {
             System.out.println("Unable to generate version.Info file: "
                     + e.getMessage());
+            if (w != null) {
```

Comment:
Findbug complains here because we use System.exit in catch block - in that case it is not guaranteed that the finally block will be executed.